.gitignore

```diff
@@ -1,2 +1,5 @@
 target
 Cargo.lock
+
```

Comment:
You can (and probably should) locally ignore these files using `git config`. Don't add special rules to the repo's .gitignore file.

Reply:
For a common IDE like Jetbrains it would be better for the repo to have the ignore. Otherwise everyone who uses a Jetbrains IDE will have to add their own ad-hoc ignore on their machine.

I agree though for more user-specific files that might show up to ignore locally, but this isn't one of those.

Reply:
We generally do not include editor-specific items in our `.gitignore`. If you wish to suggest changes to that policy, please open an issue to discuss that rather than changing it in an unrelated PR.

Reply:
Sure, I didn't know about that policy (and don't see it in CONTRIBUTING.md), but I will remove the change.